materials/section-1/src/koans/01_equalities.clj

```diff
@@ -0,0 +1,54 @@
+(ns koans.01-equalities
+  (:require [koan-engine.core :refer [mediations]]))
+
+"
+Equalities:
```

Comment:
I don't know if this is necessary considering we have the `REAME.md` in this section already.

We should also aim to get people comfortable with looking up documentation for functions anyway rather than spoon-feeding everything